exir/backend/backend_api.py

```diff
@@ -720,6 +726,8 @@ def to_backend(
             fake_edge_program = copy.deepcopy(edge_program)
         partitioner_result = partitioner_instance(fake_edge_program)
         tagged_exported_program = partitioner_result.tagged_exported_program
+        tagged_exported_program.example_inputs = edge_program.example_inputs
```

Comment:
?

Reply:
serializing here you mean exir.save, or to pte?